### DISCUSSION TITLE
Cruiser and OpenStreetMap, two great tools!

### DISCUSSION THREAD

**Discussion by ramondevic** [devemux86/cruiser#229] [Q&A] [answered]
I have been traveling by motorbike for 3 months from Catalonia to Turkey, Iran and Iraq and everything is perfect with the combination of Mapsforge's Openstreetmaps and the Cruiser app.

I prepared several routes and tracks with https://kurviger.de, exported them in Kurviger format and imported when needed in Cruiser (route and cover) and the result has been great, everything I expected actually I was able to do it, there wasn't a road missing (well, maybe some if the route had been changed and no one had informed Openstreetmaps) but the overall result was magnificent, no problem.

The entire trip was without having to use the Internet, the ruggedized phone I was carrying had no connection and in the case of having to improvise a route from A to B I did it from another phone that had data and then I was exporting to the phone that was using GPS, one day I had problems with the export, I opened a ticket on the forum and in less than an hour @devemux86 solved it, great!

I have only words of thanks for these great tools, for @devemux86 (the developer of the app) and all the users who are constantly testing and improving Cruiser.

Thank you all !

**Comment by devemux86** [comment, marked as answer]
@ramondevic thanks for the kind words and support!

Looks like you traveled a long and extraordinary journey, well done! :+1: 

If you need offline routing, please check the new detailed BRouter profile options inside Cruiser:
- https://github.com/devemux86/cruiser/discussions/175#discussioncomment-6183545

You can also expect many new and interesting features in the near future:
- https://github.com/devemux86/cruiser/discussions/9#discussioncomment-6249887
- https://github.com/devemux86/cruiser/discussions/209#discussioncomment-6278789

**Reply by ramondevic** [reply]
To calculate offline routes while traveling I used Graphhoper and tweaked the routes by adding wayoints or shapepoints to adjust the route to my liking.

Now I've arrived and I'm trying BRouter, it's much more powerful and configurable but it's about testing and fine-tuning preferences and it's more complicated to understand because sometimes one route worked and not another. It takes time and patience to try ;)

Very good job.

**Reply by devemux86** [reply]
@ramondevic 

I will improve the BRouter integration in Cruiser more and more with each new version.
It is extremely powerful, we can do anything we can imagine, like the new detailed avoidances.

We also expect many new interesting features from BRouter, see here:
- https://github.com/devemux86/cruiser/discussions/15#discussioncomment-6078026
